src/Server/bootstrapSharify.ts

```diff
@@ -84,6 +83,7 @@ export const bootstrapSharify = () => {
       "SEGMENT_AMP_WRITE_KEY",
       "SEGMENT_WRITE_KEY",
       "SENTRY_PUBLIC_DSN",
+      "SESSION_ID",
```

Comment:
I was surprised this wasn't already in here. I know we have access to it after we the app boots (in this use case its needed before boot), but I couldn't find exactly where that happens.

Reply:
It looks like `SESSION_ID` is added in this [bootstrapSharifyAndContextLocalsMiddleware middleware](https://github.com/artsy/force/blob/7b4ca6cc8a004323022549c39ec25cd87829c5a9/src/Server/middleware/bootstrapSharifyAndContextLocalsMiddleware.ts#L23-L30), executed upon each request whereas `bootstrapSharify` is [only executed once](https://github.com/artsy/force/blob/7b4ca6cc8a004323022549c39ec25cd87829c5a9/src/middleware.ts#L57) upon server initialization so I don't think this addition will have any effect as we never seed `SESSION_ID` from the process environment.